stestr/commands/run.py

```diff
@@ -318,19 +334,32 @@ def run_command(config='.stestr.conf', repo_type='file',
         if ids.find('/') != -1:
             root = ids.replace('.py', '')
             ids = root.replace('/', '.')
-        run_cmd = 'python -m subunit.run ' + ids
+        python = 'python'
+        if coverage:
+            package = 'stestr'
+            options = "--source %s --parallel-mode" % package
+            python = ("coverage run %s" % options)
+            if omit:
+                omit = "--omit=%s" % omit
+        run_cmd = ('%s -m subunit.run ' % python) + ids
 
         def run_tests():
             run_proc = [('subunit', output.ReturnCodeToSubunit(
                 subprocess.Popen(run_cmd, shell=True,
                                  stdout=subprocess.PIPE)))]
-            return load.load(in_streams=run_proc,
-                             subunit_out=subunit_out,
-                             repo_type=repo_type,
-                             repo_url=repo_url, run_id=combine_id,
-                             pretty_out=pretty_out,
-                             color=color, stdout=stdout, abbreviate=abbreviate,
-                             suppress_attachments=suppress_attachments)
+
+            result = load.load(in_streams=run_proc,
+                               subunit_out=subunit_out,
+                               repo_type=repo_type,
+                               repo_url=repo_url, run_id=combine_id,
+                               pretty_out=pretty_out,
+                               color=color, stdout=stdout,
+                               abbreviate=abbreviate,
+                               suppress_attachments=suppress_attachments)
+            if coverage:
+                os.system("coverage combine")
+                os.system("coverage html -d ./cover %s" % omit)
```

Comment:
I don't think that we want to generate html output like this from a CLI tool. If we're going to generate the report (and not just collect coverage)  I think we should call coverage report to print to stdout instead.